core/src/test/java/org/infinispan/jmx/CacheContainerStatsMBeanTest.java

```diff
@@ -75,6 +76,9 @@ public void testClusterStats() throws Exception {
 
       cache1.remove("a1");
 
+      // Wait 1 second for the cached stats to expire
+      TestingUtil.sleepThread(1000);
```

Comment:
You should replace the TimeService with a ControlledTimeService to avoid actually sleeping for 1s